lib/x11regressiontest.pm

```diff
@@ -481,9 +481,22 @@ sub exit_firefox() {
 }
 
 sub start_gnome_settings {
-    send_key "super";
-    wait_still_screen;
-    type_string "settings";
+    my $is_sle_sp1 = check_var('VERSION', '12-SP1');
+    my $workaround_repetitions = 3;
+
+    # the loop is a workaround for SP1: bug in launcher. Sometimes it doesn't react to click
+    do {
+        if ($is_sle_sp1) {
```

Comment:
again, weird name. is "sle_sp1" the same as 11 SP1? No, it isn't so better `sle12sp1` or even better use comparison functions because in general that would probably also apply to older versions, so e.g. `check_var('DISTRI', 'sle') && !sle_version_at_least('12-SP2')`